frontends/web/src/routes/account/send/send.tsx

```diff
@@ -161,26 +149,22 @@ class Send extends Component<Props, State> {
       signConfirm(() =>
         this.setState({ signConfirm: true })
       ),
-      syncdone((code) => {
-        if (this.props.code === code) {
```

Comment:
I think you should keep this condition somehow, haven't tested but maybe `this.props.account.code === code`  or similar?

Reply:
changed it in first push, also rebased in second push.